Python/aot_ceval_jit.c

```diff
@@ -729,7 +729,7 @@ static void emit_dec_qword_ptr(Jit* Dst, void* ptr, int can_use_tmp_reg) {
 }
 static void emit_inc_qword_ptr(Jit* Dst, void* ptr, int can_use_tmp_reg) {
     if (IS_32BIT_VAL(ptr)) {
-        | inc qword [&ptr]
+        | inc qword [ptr]
```

Comment:
nice catch!
this is easily reproducible: `SHOW_JIT_STATS=1 make nbody_unopt` crashed before don't know how I missed that... :(